### DISCUSSION TITLE
undefined method `in_app_purchases' for #<Spaceship::Portal::App:0x00007f904c381b00> (NoMethodError)

### DISCUSSION THREAD

**Discussion by sandy-boop** [fastlane/fastlane#20547] [General]
I do the function of uploading the purchase items in apple in batches. I hope you can help me with the following problems. Thank you.
I installed your updates and made changes

rubyCode = rubyCode + \
                'Spaceship::ConnectAPI.login("' + account + '", "' + password +'")\n' \
                'app = Spaceship::App.find("' + packageName +'")\n'

app.in_app_purchases.create!(
type: Spaceship::ConnectAPI::IAPType::{iapType},
)
 
But when I run the script, I report the following error
undefined method `in_app_purchases' for #<Spaceship::Portal::App:0x00007f904c381b00> (NoMethodError)

**Comment by jmgeffroy** [comment]
Hi, did you get a solution for that? I'm pretty new to Fastlane, and I'm facing the exact same issue...

**Reply by sandy-boop** [reply]
Hello, I can't solve that problem. I now use the API provided by Apple to write a python3 script to achieve the same function.
